index.html

```diff
@@ -1,53 +1,124 @@
+Dafny makes rigorous verification an integral part of development thus reducing costly late-stage bugs that may be missed by testing.
+</p>
+</td>
+<td style="width:40%">
+<b>Quick Links:</b>
+<ul>
+<li> <a href="https://github.com/dafny-lang/dafny/wiki/INSTALL"><b>Installation</b></a> 
+     (or a <a href="https://marketplace.visualstudio.com/items?itemName=dafny-lang.ide-vscode">VSCode plugin for Dafny</a>)
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef">Dafny Reference Manual and User Guide</a>
+<li> <a href="https://dafny.org/dafny/toc">Dafny Resources for Users</a>
+<li> <a href="https://github.com/dafny-lang/dafny">Dafny GitHub project (for developers of the Dafny tools themselves)</a>
+</ul>
+</td>
+
+
+</tr>
+</table>
+
+<table style="width:100%">
+<td style="width:65%" valign=top>
+<p>
+In addition to a verification engine to check implementation against specifications, the Dafny ecosystem includes several compilers, plugins for common software development IDEs, a LSP-based Language Server, a code formatter, a reference manual, tutorials, power user tips, books, the experiences of professors teaching Dafny, and the accumulating expertise of industrial projects using Dafny.
+Example Dafny code in a Visual Studio IDE is shown to the right.
+
+</p>
+
+<p>Dafny has support for common programming concepts such as </p>
+<ul>
+<li> mathematical and bounded integers and reals, bit-vectors, classes, iterators, arrays, tuples, generic types, refinement and inheritance
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-inductive-datatypes">inductive datatypes</a> that can have methods and are suitable for pattern matching,
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-co-inductive-datatypes">lazily unbounded datatypes</a>,
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-subset-types">subset types</a>, such as for bounded integers,
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-lambda-expressions">lambda expressions</a> and functional programming idioms,
+<li> and <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-collection-types">immutable and mutable data structures</a>.
+</ul>
+
+<p>Dafny also offers an extensive toolbox for mathematical proofs about software, including</p>
+<ul>
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-forall-statement">bounded and unbounded quantifiers</a>,
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-calc-statement">calculational proofs</a> and ability to use and prove lemmas,
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-specification-clauses">pre- and post-conditions, termination conditions, loop invariants, and read/write specifications</a>
+</ul>
+
+</td>
+
+<td style="width:35%" img-align:center>
+<img src="Banner/Demo.png" alt="Demo code" width="400" height="600">
```

Comment:
That aspect ratio is too narrow.

Reply:
+1 on this - the screen shot is nice but definitely squished.